test/jdk/java/foreign/TestArrays.java

```diff
@@ -41,47 +40,55 @@
 import java.util.function.Consumer;
 import java.util.function.Function;
 
+import jdk.incubator.foreign.ValueLayout;
 import org.testng.annotations.*;
 
+import static jdk.incubator.foreign.ValueLayout.JAVA_BYTE;
+import static jdk.incubator.foreign.ValueLayout.JAVA_CHAR;
+import static jdk.incubator.foreign.ValueLayout.JAVA_DOUBLE;
+import static jdk.incubator.foreign.ValueLayout.JAVA_FLOAT;
+import static jdk.incubator.foreign.ValueLayout.JAVA_INT;
+import static jdk.incubator.foreign.ValueLayout.JAVA_LONG;
+import static jdk.incubator.foreign.ValueLayout.JAVA_SHORT;
 import static org.testng.Assert.*;
 
 public class TestArrays {
 
     static SequenceLayout bytes = MemoryLayout.sequenceLayout(100,
-            MemoryLayouts.JAVA_BYTE
+            JAVA_BYTE
     );
 
     static SequenceLayout chars = MemoryLayout.sequenceLayout(100,
-            MemoryLayouts.JAVA_CHAR
+            ValueLayout.JAVA_CHAR
```

Comment:
```suggestion
            JAVA_CHAR
```
And for other usages.